build/app_template_files/__NativeSwiftTemplateAppName__/__NativeSwiftTemplateAppName__/Info.plist

```diff
@@ -15,7 +15,7 @@
         <key>CFBundleIdentifier</key>
         <string>__CompanyIdentifier__.${PRODUCT_NAME:rfc1034identifier}</string>
         <key>CFBundleInfoDictionaryVersion</key>
-        <string>6.0</string>
+        <string>8.0</string>
```

Comment:
You don't want to bump this number.  It's actually a separate version number specifying the plist schema version.  6.0 is the latest.